libraries/postgres/src/Database.js

```diff
@@ -109,9 +108,11 @@ class Database {
           try {
             let res;
             if (hasNamedArguments) {
-              const placeholders = Object.keys(args[0]).map(arg => `:${arg}`).join(',');
-              const { text, values } = named(`select * from "${method.name}"(${placeholders})`, { useNullForMissing: true })(args[0]);
-              res = await client.query(text, values);
+              const obj = args[0];
+              const keys = Object.keys(obj);
+              const values = keys.map(k => obj[k]); // order must match keys!
+              const placeholders = keys.map((key, i) => `${key} => $${i + 1}`).join(',');
```

Comment:
could you inject something by naming a key in a certain way here?

Reply:
Yes, good point, although I'd expect those not to come from user input.  But just in case, I can add a somewhat more restrictive regex where we check for the `_in` suffix.

Reply:
ok great. I figured it would be fine for now but who knows what future-us will end up doing with this